frontend_app_file/src/component/Kanban.jsx

```diff
@@ -0,0 +1,386 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import { v4 as uuidv4 } from 'uuid'
+import Board from '@lourenci/react-kanban'
+import '@lourenci/react-kanban/dist/styles.css'
+import {
+  IconButton,
+  PromptMessage,
+  handleFetchResult,
+  putFileContent,
+  removeLocalStorageItem,
+  getLocalStorageItem,
+  setLocalStorageItem
+} from 'tracim_frontend_lib'
+
+require('./Kanban.styl')
+
+function moveElem (array, fromPosition, toPosition, elem) {
+  const a = array.slice()
+  a.splice(fromPosition, 1)
+  a.splice(toPosition, 0, elem)
+  return a
+}
+
+export default translate()(class Kanban extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      loading: false,
+      mustSave: false,
+      board: {
+        columns: [],
+        ...JSON.parse(props.content.raw_content || '{}')
+      },
+      isDrafAvailable: !!getLocalStorageItem(
+        'rawContent',
+        props.content,
+        'kanban'
```

Comment:
this label should exist as appName like other apps do

Reply:
Yes, I planned to do this when we agree that this is the right design. Moreover, we are in app file so we need to discuss how to handle this case. This is not really an appName, but more a "local storage slot"

Reply:
indeed